styles/component-styles/alert.css

```diff
@@ -19,29 +6,29 @@
 }
 
 .alert-error{
-    background-color: var(--error-background);
+    background-color: var(--error-background-color);
     padding: 1rem;
     border-radius: 1rem;
-    border: 2px solid var(--error-border);
+    border: 2px solid var(--error-border-text-color);
 }
 
 .alert-warning{
-    background-color: var(--warning-background);
+    background-color: var(--warning-background-color);
     padding: 1rem;
     border-radius: 1rem;
-    border: 2px solid var(--warning-border);
+    border: 2px solid var(--warning-border-text-color);
 }
 
 .alert-info{
-    background-color: var(--info-background);
+    background-color: var(--info-background-color);
     padding: 1rem;
     border-radius: 1rem;
-    border: 2px solid var(--info-border);
+    border: 2px solid var(--info-border-text-color);
 }
 
 .alert-success{
-    background-color: var(--success-background);
+    background-color: var(--success-background-color);
     padding: 1rem;
     border-radius: 1rem;
```

Comment:
if you're giving padding and radius same to all alerts you can add a separate class for common styling

Reply:
Good observation Zuber. made a separate class for margin and padding.